requirements.txt

```diff
@@ -2,3 +2,4 @@ pyyaml
 jinja2
 minify_html
 openpyxl
+jsonschema
```

Comment:
That new requirement only applies to the test. Should we have different requirement for the test and the actual software ?